plugins/newsdownloader.koplugin/main.lua

```diff
@@ -47,13 +47,7 @@ function NewsDownloader:addToMainMenu(menu_items)
         if not lfs.attributes(NEWS_DL_DIR, "mode") then
             lfs.mkdir(NEWS_DL_DIR)
         end
-
-        FEED_CONFIG_PATH = NEWS_DL_DIR .. FEED_CONFIG_FILE
-        if not lfs.attributes(FEED_CONFIG_PATH, "mode") then
-            logger.dbg("NewsDownloader: Creating init configuration")
-            FFIUtil.copyFile(FFIUtil.joinPath(self.path, FEED_CONFIG_FILE),
-                             FEED_CONFIG_PATH)
-        end
+	FEED_CONFIG_PATH = NEWS_DL_DIR .. FEED_CONFIG_FILE
```

Comment:
Also generally `ALL_CAPS` means globals while I assume these are meant to be locals. I don't have time to look at in detail but they should probably be lowercase?